taskgenerator.js

```diff
@@ -0,0 +1,69 @@
+const schemas = require('./schemas');
+const Task = schemas.task;
+const Image = schemas.image;
+        
+
+function sleep(milliseconds) {
+    return new Promise(resolve => setTimeout(resolve, milliseconds));
+}
+
+async function insertTaskPeriodic(task, timeInMs) {
+    while (true) {
+
+        Task.findOne({title: task.title}, (err, doc) => {
+            if(err) return console.error(err);
```

Comment:
very little commentary, could for example be: // check whether task is already in database, if no task is found, save the task to db; like in function createLabelImageTasksHourly